internal/pkg/cli/deploy/env.go

```diff
@@ -154,8 +160,9 @@ func NewEnvDeployer(in *NewEnvDeployerInput) (*envDeployer, error) {
 		env: in.Env,
 
 		templateFS:       template.New(),
-		s3:               s3.New(envManagerSession),
+		s3:               awss3.New(envManagerSession),
 		prefixListGetter: ec2.New(envRegionSession),
+		bucketNameGetter: s3.New(envManagerSession),
```

Comment:
(just brainstorming 🧠 ⚡ ):

Seeing these two field names, it feels odd that we would need to initialize one client for s3 and another for getting a bucket name 💭 

I wonder if instead our "higher"-level packages like `internal/pkg/s3` should _embed_ the `internal/pkg/aws/s3`  clients so that they strictly augment the AWS SDK's method and we don't have to import two pkgs?

Reply:
Do you mean all the public things in `aws/s3` should be embedded in `s3`? How about public function/variables?